pkg/sql/vars.go

```diff
@@ -702,16 +702,7 @@ var varGen = map[string]sessionVar{
 	// See https://www.postgresql.org/docs/10/static/runtime-config-client.html#GUC-TIMEZONE
 	`timezone`: {
 		Get: func(evalCtx *extendedEvalContext) string {
-			// If the time zone is a "fixed offset" one, initialized from an offset
```

Comment:
(this looks already broken, so i will fix this in a separate PR - if you have a better name idea though lmk!)

Reply:
That's fine. But why change it in this PR?

Reply:
I needed this function here: https://github.com/cockroachdb/cockroach/pull/42376/files#diff-4fd6eaee39da7af19c97fbf9311f2c1fR1832